### ISSUE TITLE
[ion | Settings] Update System Idea

### ISSUE THREAD

**Issue by devdl11** [UpsilonNumworks/Upsilon#44]
I was wondering, with the existence of the dump memory system, if it would not be interesting to update the calculator from another calculator, to transfer the version. It's just an idea, tell me what you think (I'm ready to make it happen)

It would allow to update the calculators more easily by having only a usb> micro usb adapter and a cable ^^

**Comment by Lauryy06** [comment]
Yes, this can be an interesting feature, probably rarely used, but a  really interesting one.

**Comment by FaustinM** [comment]
A killer feature is to share python script to another calculator with a MicroUSB to MicroUSB cable

**Comment by Yaya-Cout** [comment]
I think Zardam started to implement this in https://github.com/zardam/epsilon/tree/uart_over_usb
